cfme/tests/services/test_provision_stack.py

```diff
@@ -26,73 +26,127 @@
 AWS_TEMPLATE = """
```

Comment:
**OPTIONAL:** This is getting big enough that it should probably be an external file now, ie not part of the test file. We should make an issue to do this.